52/vipinreyo/pomodoro.py

```diff
@@ -0,0 +1,80 @@
+"""
+A simple command line based Pomodoro timer. Timer duration and break duration can be passed as arguments, but are
+optional. Default timer value is set as 25 minutes. Default break duration is set as 0 minutes.
+The timer is defined as a class for reuse.
+
+Note: Please pip install 'colorama' pacakge as this is used to color the output.
+"""
+
+from datetime import datetime, timedelta
+import time
+import sys
+import argparse
+from colorama import Fore
+import itertools
+
+NOTIFICATION_COUNT = 5
+SECONDS = 60
+DEFAULT_POMODORO_DURATION = 25
+DEFAULT_BREAK_DURATION = 0
+
+
+class PomodoroTimer:
+
+    def __init__(self, work_duration, break_duration):
+        self.work_minutes = work_duration
+        self.break_minutes = break_duration
+
+    def start_timer(self):
+
+        td = td_duration = timedelta(minutes=self.work_minutes)
+        timer = datetime.now().replace(microsecond=0) + td
+        timedelta_1s = timedelta(seconds=1)
+        timer_loop = True
+
+        timer_msgs1 = itertools.cycle(['Timer expired!', ' '])
+
+        # Timer loop
+        while timer_loop:
+            # Print the timer
+            sys.stdout.write(Fore.BLUE + f'\r {int(td.seconds/60):02}:{(td.seconds % 60):02}')
+            sys.stdout.flush()
```

Comment:
cool, need to run this locally but I assume this will refresh my terminal output

Reply:
yes, that's right Bob. Cheers